__mocks__/patient-note-config.mock.ts

```diff
@@ -0,0 +1,10 @@
+export const ConfigMock = {
+  notesConfig: {
```

Comment:
Oh, sorry @usamaidrsk  I should have noted that when moving this we can now rely on the EMR API properties configuration endpoint to fetch these values:

/openmrs/ws/rest/v1/emrapi/configuration

The constants you've laid out below should map to the following properties in the configuration REST response

"clinicanEncounterRole" -> "clinicianEncounterRole.uuid" 
"encounterTypeUuid" -> "visitNoteEncounterType.uuid"
"encounterNoxtTextConceptUuid" -> "consultFreeTextCommentsConcept.uuid"

And I don't think we need the other three properties (at least not yet)
"visitDiagnosesConceptUuid"
"problemLIstConceptUuid"
"formConceptUuid"